.github/workflows/sleeper.yml

```diff
@@ -28,24 +28,24 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
-      - name: setup python environment
-        uses: actions/setup-python@v1.1.1
+      - name: setup python environmentactions/setup-python@v2
```

Comment:
revert that line? find/replace that went wrong?

Reply:
It also looked weird to me, I took it from the jupyters one. Reverted, thanks.